src/components/Container.js

```diff
@@ -179,6 +186,10 @@ class Container extends React.Component {
       hydrantSelected, openManualAssignment,
       manualAssignmentItems, manualAssignmentItemsAdded, focusedHydrant
     } = this.props;
+
```

Comment:
I'm not 100% sure whether this is in the linter, but in general I think one blank line is usually enough to separate things for readability.